### PR TITLE
Benp/jshint violations

### PR DESCRIPTION
**Pull request by benpatterson**
@Bachmann1234 mind taking a look?

### CODE REVIEW

**Comment by Bachmann1234** [comment]
Ill take a look today or tomorrow!

**Comment by benpatterson** [comment]
yo @Bachmann1234 looks like I have broken tests :( I'll address those and let you know when it's GTG for you. Sorry about that...

File: diff_cover/tests/test_violations_reporter.py [hunk 1]
```diff
@@ -976,7 +976,162 @@ def test_quality_pregenerated_report_continuation_char(self):
         self.assertEqual(violations, [Violation(2, u"W1401: Invalid char '\ufffd'")])
 
 
+class JsHintQualityReporterTest(unittest.TestCase):
+
+    def setUp(self):
+
```
Comment (Bachmann1234) [inline diff_cover/tests/test_violations_reporter.py]: delete this blank line. Sorry to flake8 you. Don't care about the long lines though (im looking at the diff-quality report :-P)

File: diff_cover/violations_reporter.py [hunk 2]
```diff
@@ -339,6 +344,17 @@ def _run_command(self, src_path):
 
         return stdout.strip().decode(self.STDOUT_ENCODING, 'replace')
 
+    def _run_command_simple(self, command):
```
Comment (Bachmann1234) [inline diff_cover/violations_reporter.py]: Kinda weird that this was not hit by the tests

**Comment by Bachmann1234** [comment]
A couple trivial comments. Looks pretty good to me. Can you squash this down?

File: diff_cover/violations_reporter.py (hunk 2 above)
Comment (benpatterson) [inline diff_cover/violations_reporter.py]: Yeah. I deferred to coverage.py on this, and added tests. Made sense to be explicit about the method anyway.

**Comment by benpatterson** [comment]
@Bachmann1234 Added tests, fixed up flake8, and squashed. Happy to iterate again if you've got additional comments.

**Comment by Bachmann1234** [comment]
Cool ill give this a last once over soon!

**Comment by Bachmann1234** [comment]
Lets do this! Thanks for the PR

**Comment by Bachmann1234** [comment]
https://pypi.python.org/pypi/diff_cover ITS OUT! :-) Thanks again

**Comment by benpatterson** [comment]
Awesome!!!

**Pull request merged**